src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -85,7 +85,7 @@ public int MaximumFollowRelLink
 
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-macos`
Invoke-WebRequest tests.Validate Invoke-WebRequest -OutFile
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 648
648:         $jsonContent.headers.Host | Should -Be $uri.Authority
```
Invoke-WebRequest tests.Invoke-WebRequest File Resume Feature.Error occurred in Context block
```powershell
Cannot process the argument because the value of statusDescription cannot be null or empty.
At /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1:1757 char:13
```
Invoke-RestMethod tests.Validate Invoke-RestMethod -OutFile
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2202
2202:         $jsonContent.headers.Host | Should -Be $uri.Authority
```
Invoke-RestMethod tests.Invoke-RestMethod File Resume Feature.Error occurred in Context block
```powershell
Cannot process the argument because the value of statusDescription cannot be null or empty.
At /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1:3267 char:13
```